setup.cfg

```diff
@@ -9,6 +9,8 @@ ignore =
     E402
     # line too long - let black worry about that
     E501
+    # Allow weird empty tests #TODO: Check what purpose they are serving
+    E704
```

Comment:
@TomNicholas I ignored these since they got upset by [these sort of empty tests](https://github.com/xgcm/xgcm/blob/33db3f210f67ff195b225e67c3326b8dd11ca998/xgcm/test/test_axis.py#L100-L101) that were introduced [here](https://github.com/xgcm/xgcm/commit/1188a0a24898971ce060bc9f34b79e2fcabc33f9).

Just wanted to quickly check if you remember why these empty tests are there in the first place?